src/WebJobs.Script.WebHost/Handlers/WebScriptHostHandler.cs

```diff
@@ -60,6 +59,11 @@ public WebScriptHostHandler(HttpConfiguration config, int hostTimeoutSeconds = H
                 await scriptHostManager.DelayUntilHostReady(_hostTimeoutSeconds, _hostRunningPollIntervalMilliseconds);
             }
 
+            if (StandbyManager.IsWarmUpRequest(request))
```

Comment:
When a warmup request comes in, we run some additional warmup logic here, in addition to invoking the actual warmup function, which is a full fledged C# http function.